client/src/components/QandA/QuestionList.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import QuestionListEntry from './QuestionListEntry.jsx';
+import AddQuestion from './AddQuestion.jsx';
+
+const QuestionList = (props) => {
+  return (
+    <div className="container">
+      <p className="text-start fs-5 fw-light">
+      Questions and Answers
+      </p>
+      <div className="input-group mb-3">
+        <input type="text" className="form-control" placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..." aria-label="Recipient's username" aria-describedby="button-addon2"></input>
+        <button className="btn btn-outline-secondary" type="button" id="button-addon2">Filter Questions</button>
+        {/* the above will be replaced by a magnifiying glass icon */}
+      </div>
+      <div className="row">
+        {props.data.results.map((question, index) => {
+          if (index < 2) {
```

Comment:
Rather than hardcoding the 2 here, maybe you create a variable in state that would be incremented by 2 each time the user clicks the show more button. Once the variable is >= the total number of entries, you could conditionally hide the 'show more' button....maybe...

Reply:
That sounds like a great idea. Appreciate the input!